sympy/physics/units/definitions.py

```diff
@@ -83,87 +100,133 @@
 
 # Common time units
 
-ms = millisecond = milliseconds = Quantity("millisecond", time, milli*second, "ms")
-us = microsecond = microseconds = Quantity("microsecond", time, micro*second, "us")
-ns = nanosecond = nanoseconds = Quantity("nanosecond", time, nano*second, "ns")
-ps = picosecond = picoseconds = Quantity("picosecond", time, pico*second, "ps")
-
-minute = minutes = Quantity("minute", time, 60*second)
-h = hour = hours = Quantity("hour", time, 60*minute)
-day = days = Quantity("day", time, 24*hour)
-
-anomalistic_year = anomalistic_years = Quantity("anomalistic_year", time, 365.259636*day)
+ms = millisecond = milliseconds = Quantity("millisecond", time, milli * second,
+                                           "ms")
+us = microsecond = microseconds = Quantity("microsecond", time, micro * second,
+                                           "us")
+ns = nanosecond = nanoseconds = Quantity("nanosecond", time, nano * second,
+                                         "ns")
+ps = picosecond = picoseconds = Quantity("picosecond", time, pico * second,
+                                         "ps")
+
+minute = minutes = Quantity("minute", time, 60 * second)
+h = hour = hours = Quantity("hour", time, 60 * minute)
+day = days = Quantity("day", time, 24 * hour)
+
+anomalistic_year = anomalistic_years = Quantity("anomalistic_year", time,
+                                                365.259636 * day)
 sidereal_year = sidereal_years = Quantity("sidereal_year", time, 31558149.540)
-tropical_year = tropical_years = Quantity("tropical_year", time, 365.24219*day)
-common_year = common_years = Quantity("common_year", time, 365*day)
-julian_year = julian_years = Quantity("julian_year", time, 365.25*day)
-draconic_year = draconic_years = Quantity("draconic_year", time, 346.62*day)
-gaussian_year = gaussian_years = Quantity("gaussian_year", time, 365.2568983*day)
-full_moon_cycle = full_moon_cycles = Quantity("full_moon_cycle", time, 411.78443029*day)
+tropical_year = tropical_years = Quantity("tropical_year", time,
+                                          365.24219 * day)
+common_year = common_years = Quantity("common_year", time, 365 * day)
+julian_year = julian_years = Quantity("julian_year", time, 365.25 * day)
+draconic_year = draconic_years = Quantity("draconic_year", time, 346.62 * day)
+gaussian_year = gaussian_years = Quantity("gaussian_year", time,
+                                          365.2568983 * day)
+full_moon_cycle = full_moon_cycles = Quantity("full_moon_cycle", time,
+                                              411.78443029 * day)
 
 year = years = tropical_year
 
-#### CONSTANTS ####
+# === CONSTANTS === #
 
 # Newton constant
-G = gravitational_constant = Quantity("gravitational_constant", length**3*mass**-1*time**-2, 6.67408e-11*m**3/(kg*s**2), abbrev="G")
+G = gravitational_constant = Quantity(
+    "gravitational_constant",
+    length**3 * mass**-1 * time**-2,
+    6.67408e-11 * m**3 / (kg * s**2),
+    abbrev="G")
```

Comment:
Where does this indentation convention come from? I personally prefer leaving line 135 on 134 and then indenting the next lines to the edge of the open parentheses.